docs/docs/widgets/qr-scanner.md

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 1
+---
+
+# QR Scanner
+Scan QR codes using device camera and hold the data they carry. 
+<img class="screenshot-full" src="/img/widgets/qr-scanner/qr-scanner.jpeg" alt="ToolJet - QR Scanner" height="420"/>
+
+#### Known issue: 
+In IOS, you might have to stick to the Safari browser as camera access had been restricted for third party browsers.
+
+## Properties
+#### active
```

Comment:
Are we hiding the widget when `active` is resolved as `false` or we will still show the widget in a disabled state?

Reply:
Right now it wont render anything if `active` resolves to false. I'm waiting to learn from the scenarios on which the `active` property will be made use of.